examples/energy-monitor/ui/pages/dashboard.slint

```diff
@@ -15,7 +15,10 @@ export component ValueTile {
     in property <bool> active;
 
     i-group-box := GroupBox {
-        i-value-display := ValueDisplay {  
+        preferred-width: 100%;
```

Comment:
I would think that a GroupBox should already have `preferred-width: 100%;`

Reply:
done